AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -249,6 +253,7 @@ - (void)flushQueue {
 
                // Forward logs to the sender.
                [self.sender sendAsync:container
+                            appSecret:self.appSecret
```

Comment:
Not a change request but we need to consider moving this method and others to channel group some time later.